proc/disasm_amd64.go

```diff
@@ -111,3 +112,50 @@ func (thread *Thread) resolveCallArg(inst *ArchInst, currentGoroutine bool, regs
 	}
 	return &Location{PC: pc, File: file, Line: line, Fn: fn}
 }
+
+type instrseq []x86asm.Op
+
+var windowsPrologue = instrseq{x86asm.MOV, x86asm.MOV, x86asm.LEA, x86asm.CMP, x86asm.JBE}
+var windowsPrologue2 = instrseq{x86asm.MOV, x86asm.MOV, x86asm.CMP, x86asm.JBE}
+var linuxPrologue = instrseq{x86asm.MOV, x86asm.LEA, x86asm.CMP, x86asm.JBE}
```

Comment:
Does this apply to OSX as well (I'm assuming so), if so let's just rename this `unixPrologue` for now. Also, I'm somewhat tempted to prefer splitting these declarations out into their own OS specific file, but I fear that would hide the discoverability of `prologues`.